internal/report/report.go

```diff
@@ -46,27 +46,22 @@ No custom resources
 {{ end }}
 `
 
-	operandJsonReportTemplate = `{{with $dot := .}}{{range $index, $value := .CustomResources }}
+	OperandJsonReportTemplate = `{{with $dot := .}}{{range $index, $value := .CustomResources }}
```

Comment:
If we keep the other functions, we won't need to expose these. These are an implementation detail.

Reply:
yes makes sense to keep templates internal 👍